CHANGELOG.md

```diff
@@ -8,15 +8,22 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 
 ### Added
+- Connector class for retrieving ORCID data by ID or name
+- methods to transform from OcidPerson to mex person
 
 ### Changes
-
+- added validation_alias to orcid url in `settings.py`
+- remodeled OrcidPerson in `persony.py`
+- using `OrdicPerson`'s orcidid.uri for transformation instead stitiching it together
+- fetch method in OrcidConnector accepts directly filters + passes query als param to search-endpoint
+- moved `get_data_by_id` from extract to connector
 ### Deprecated
 
 ### Removed
-
+- ``mapToOrcidPerson`` and `reduce_metadata` deleted. Replaced with model.validate fct
```

Comment:
these changelog entries are not quite descriptive of what this PR will do. these are steps you took during development, but they do not describe the diff of this branch compared to main.

example: `remodeled OrcidPerson in persony.py` -> this class doesn't exist on main, so this PR doesn't _remodel_ it, it will _add_ it.